R/gcloud-utils.R

```diff
@@ -41,12 +41,15 @@ scope_deployment <- function(application = getwd(), config) {
 
   # default excludes plus any additional excludes in the config file
   config <- cloudml::config(config = config)
-  exclude <- c("local", "jobs")
+  exclude <- c("local", "jobs", ".git", ".svn")
```

Comment:
Is the intention for this to ensure `.git` directories in subdirectories aren't included? I'm not sure if this works since the pattern generated by `glob2rx(".git")` is just "^\\.git$".